### DISCUSSION TITLE
sh_fmt: semicolon formatting

### DISCUSSION THREAD

**Discussion by KindDragon** [mvdan/sh#994] [Q&A]
shfmt reformat this
```
HOST_SRC="$(cd $(dirname $0)/..;pwd)"
SCRIPT_PATH="$(cd $(dirname $0);pwd)"
```
like this (even with -s)
```
HOST_SRC="$(
        cd $(dirname $0)/..
        pwd
)"
SCRIPT_PATH="$(
        cd $(dirname $0)
        pwd
)"
```
Can I force shfmt to use single line formatting for such cases?

**Comment by mvdan** [comment]
See https://github.com/mvdan/sh/issues/261 which is closed. We only plan to allow this in a few select cases like https://github.com/mvdan/sh/issues/564 or https://github.com/mvdan/sh/issues/679.
